core/src/main/scala/epollcat/internal/ch/EpollAsyncServerSocketChannel.scala

```diff
@@ -131,7 +134,11 @@ final class EpollAsyncServerSocketChannel private (fd: Int)
       handler: CompletionHandler[AsynchronousSocketChannel, _ >: A]
   ): Unit = {
     if (readReady) {
-      val clientFd = syssocket.accept4(fd, null, null, EpollAsyncSocketChannel.SOCK_NONBLOCK)
+      val clientFd =
+        if (LinktimeInfo.isLinux)
+          socket.accept4(fd, null, null, SOCK_NONBLOCK)
+        else
+          socket.accept(fd, null, null)
```

Comment:
It seems the SN binding for `accept` is still bugged with `null` arguments, so I call our own binding here.

Reply:
> It seems the SN binding for accept is still bugged with null arguments,

That is both news and a disappointment.  I will look into that in daylight.
This almost surely falls into the category of code that I got rid of for 0.5.0,
but that does us no good for 0.4.n.

I am surprised by the `null` arguments. I can trace the code, but I would expect
the enclosing method to actually use the accepted sockaddr to fill in the 
Socket structure returned.  I need to look at more than a four line window.
All things in time.